test/instrumentation/modules/@elastic/elasticsearch.test.js

```diff
@@ -7,7 +7,8 @@ const agent = require('../../../..').start({
   metricsInterval: 0,
```

Comment:
Review Note: Span Compression can effect _when_ a span is recorded, which can create race-like conditions in the existing test suites when enabled.  To counter this, ` spanCompressionEnabled: false` has been set in these tests.